rc-events-service/src/main/scala/com/epam/random_coffee/events/services/impl/Doobie_support.scala

```diff
@@ -0,0 +1,10 @@
+package com.epam.random_coffee.events.services.impl
+
+import com.epam.random_coffee.events.model.EventId
+import doobie.Meta
+
+trait Doobie_support {
```

Comment:
Please, follow naming conventions
```suggestion
trait DoobieSupport {
```

Reply:
fixed